test/powershell/Modules/Microsoft.PowerShell.Utility/Clear-Variable.Tests.ps1

```diff
@@ -35,7 +35,7 @@ Describe "Clear-Variable DRT Unit Tests" -Tags "CI" {
 		Set-Variable foo bar
 		&{
 			Set-Variable foo baz
-			$foo | should be baz
+			$foo | Should -Be baz
```

Comment:
For strings, unless we explicitly want case insensitive matching, we should use `-BeExactly`